pepper-apis/docs/specification/src/endpoints/user.studies.activities.yml

```diff
@@ -17,7 +17,7 @@ get:
   tags:
     - Activities
   summary: summary of a participant's activities
-  description: returns a summary for each activity instance a `user` has created for a specific `study`
+  description: returns a summary for each activity instance a `user` has for a specific `study`, excluding ones that are hidden
```

Comment:
Most documentation changes are updates to reflect _current_ behavior, with a few additions for documenting "hidden" instances.